_layouts/doc.html

```diff
@@ -64,18 +64,17 @@
 						<a class="menu-group-title" href="{{ site.baseurl }}/docs/ui-toolkit/components/typography">Components</a>
 						<ul id="ui-toolkit-components" class="menu-group-wrapper level-3">
 							<li><a href="{{ site.baseurl }}/docs/ui-toolkit/components/typography">Typography</a></li>
-							<li><a href="{{ site.baseurl }}/docs/ui-toolkit/components/navigation-bar">Navigation Bar</a></li>
-							<li><a href="{{ site.baseurl }}/docs/ui-toolkit/components/dropdown-menu">Dropdown Menu</a></li>
-							<li><a href="{{ site.baseurl }}/docs/ui-toolkit/components/list-view">List View</a></li>
-							<li><a href="{{ site.baseurl }}/docs/ui-toolkit/components/grid-view">Grid View</a></li>
+							<li><a href="{{ site.baseurl }}/docs/ui-toolkit/components/navigation-bar">NavigationBar</a></li>
+							<li><a href="{{ site.baseurl }}/docs/ui-toolkit/components/dropdown-menu">DropdownMenu</a></li>
```

Comment:
If you decided to remove the space, components need to be named the same as [here](https://github.com/shoutem/ui/blob/develop/components/DropDownMenu.js).